### DISCUSSION TITLE
teoria relazionale - 1

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#68] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/f5f8fc19-a75f-4e31-b3fc-344a1b71e09e)

**Comment by IonutCicio** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/4daacd1f-27eb-41fe-b8a6-0e212e0164bf)

**Reply by CiottoloMaggico** [reply]
Forse mi perdo qualcosa io ma l'AD finale da dove salta fuori? in una decomposizione ci deve essere una relazione che corrisponde alla chiave?

**Reply by IonutCicio** [reply]
Ci deve essere una relazione che contiene una chiave, altrimenti ci potrebbe essere un **join con perdita**. Non lo abbiamo verificato / dimostrato, quindi prendilo per buono.

**Reply by CiottoloMaggico** [reply]
ah ok, non ho seguito le lezioni e sto leggendo dalle slide, non lo sapevo. Grazie mille

**Reply by CiottoloMaggico** [reply]
@CuriousCI l'algoritmo di decomposizione sulle slide non mi sembra tenga conto di questo caso, mi sto sbagliando io ?

**Reply by IonutCicio** [reply]
Hai ragione, però è specificato alla slide 7 del pdf 19

![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/3d2237ec-0506-40be-9b83-6e18d65df4d4)

**Reply by alemilos** [reply]
Come funziona la regola dell'intersezione ? Non capisco come hai scelto i diversi insiemi
